cflib/crtp/radiodriver.py

```diff
@@ -641,6 +649,11 @@ def run(self):
                 else:
                     waitTime = 0
 
+            # If there is a rate limit setup, sleep here to force the rate limit
+            if self.rate_limit:
+                time.sleep(1.0/self.rate_limit)
```

Comment:
Just a thought, nowhere in the code does anything state that `rate_limit` is in Hz ... should this be more explicit?

Reply:
An SI rate is normally Hz so I did not think about precising it. Are you envisioning a comment or renaming the variable with "_hz" in the name?

Reply:
I am not sure, perhaps just some cognitive help while reading this part:

```Python
            # If there is a rate limit setup, sleep here to force the rate limit (Hz)
            if self.rate_limit:
                time.sleep(1.0/self.rate_limit)
```